README.md

```diff
@@ -1 +1,37 @@
-# youtube_project
\ No newline at end of file
+# Youtube Project
+
+> This is the first project of the Microverse Program, which focus on applying CSS and Html knowledge to recreate a Youtube webpage.
+
+![screenshot](./images/youtube.png)
+
+Additional description about the project and its features:
+
+## Built With
+
+- Html and CSS.
+
+## Live Demo
+
+[live demo link](https://abouhid.github.io/youtube_project/)
```

Comment:
### Live demo link is not updated with the applied changes from the previous reviewer.